aya/src/programs/tc.rs

```diff
@@ -99,17 +105,24 @@ impl TcAttachType {
     }
 }
 
+/// Options for SchedClassifier attach
+#[derive(Default)]
+pub struct TcOptions {
+    /// `priority`: priority assigned to tc program with lower number = higher priority.
```

Comment:
No need to repeat the field name, see https://deploy-preview-418--aya-rs-docs.netlify.app/user/aya/programs/tc/struct.tcoptions (the first comment in the PR has a preview of docs including your changes, which is super helpful to check that everything looks ok)

Reply:
```
/// The priority the program is attached with.
///
/// Lower value means higher priority. If set to `0` the system chooses the next
/// highest priority or `0xC000` when attaching the first filter.
pub priority: u16,
/// Unique identifier for a program at a given priority level. If set to `0`,
/// the system chooses a handle.
pub handle: u32,
```

Reply:
Done